capa/features/__init__.py

```diff
@@ -187,7 +209,10 @@ def __init__(self, regex, match):
         self.match = match
 
     def __str__(self):
-        return 'regex(string =~ %s, matched = "%s")' % (self.value, self.match)
+        return 'regex(string =~ %s, matched = "%s")' % (self.value, self.get_match_str().replace('"', '\\"'))
```

Comment:
instead of doing `replace...` at various locations, maybe use `escape_string` or another helper?